docs/source/diagrams/ReflectometryReductionOne_HighLvl-v1_wkflw.dot

```diff
@@ -12,37 +12,31 @@ subgraph params {
 subgraph decisions {
  $decision_style
   checkXUnit      [label="X axis in &lambda;?"]
-  checkTransRun   [label="FirstTransmissionRun\ngiven?"]
   checkThetaIn    [label="ThetaIn given?"]
 }
 
 subgraph algorithms {
  $algorithm_style
-  divideTrans   [label="Divide by\ntransmission run"]
-  corrAlg       [label="Apply correction algorithm\nif specified"]
   calcTheta     [label="Calculate ThetaIn"]
   corrDetPos    [label="Correct detector\npositions if necessary"]
   convertMom    [label="Convert to momentum\ntransfer, rebin and scale"]
 }
 
```

Comment:
The high level diagram looks very good now with the last commit. However, could we, just for consistency, define the above as `processes`? They are not algorithms any more.